source/Sashimi.Terraform/ActionHandler/TerraformVariableFileGenerator.cs

```diff
@@ -112,10 +112,10 @@ static string GetHclVariableValue(string? propertyType, JProperty property)
         {
             if (propertyType != null && propertyType.StartsWith(TerraformDataTypes.RawPrefix))
             {
-                return "\"" + HclParser.EscapeString(property.Name) + "\" = " + property.Value;
+                return HclParser.EscapeString(property.Name) + " = " + property.Value;
```

Comment:
In Terraform > 0.12 the parameter names cannot be surrounded with double quotes.
I tested this change with 0.11 and it works as well